.taskcluster.yml

```diff
@@ -0,0 +1,30 @@
+version: 0
+metadata:
+  name: "taskcluster-lib-monitor test"
+  description: "tests for metrics and monitoring library"
+  owner: "{{ event.head.user.email }}"
+  source: "{{ event.head.repo.url }}"
+
+tasks:
+  - provisionerId: "{{ taskcluster.docker.provisionerId }}"
+  workerType: "{{ taskcluster.docker.workerType }}"
+  extra:
+    github:
+      env: true
+      events:
+        - pull_request.opened
+        - pull_request.synchronize
+        - pull_request.reopened
+        - push
+  payload:
+    maxRunTime: 3600
+    image: "quay.io/mrrrgn/ubuntu-ci:0.0.1"
```

Comment:
Unfortunately this line won't be very good anymore (I need to update the docs to reflect this!).